lib/Sema/CSGen.cpp

```diff
@@ -3431,6 +3431,17 @@ class InferUnresolvedMemberConstraintGenerator : public ConstraintGenerator {
     return Expr->getType();
   }
 
+  Type visitDeclRefExpr(DeclRefExpr *Expr) override {
```

Comment:
I think we should review what `InferUnresolvedMemberConstraintGenerator` is supposed to do and maybe instead just expect the number of callbacks we have for constraint generation...